scene/resources/texture.cpp

```diff
@@ -406,6 +416,7 @@ void PortableCompressedTexture2D::_set_data(const Vector<uint8_t> &p_data) {
 	}
 
 	image_stored = true;
+	size_override = size; // Set the same size or do not set size override with uninitialized value!
```

Comment:
You can remove this comment, but handle size_override anyway. Because this is produce an error if it uninitialized (Vector2i.Zero)